runtime/gc_vlhgc/MemorySubSpaceTarok.hpp

```diff
@@ -60,6 +60,8 @@ class MM_MemorySubSpaceTarok : public MM_MemorySubSpace
 
 	MM_HeapRegionManager *_heapRegionManager;	/**< Stored so that we can resolve the table descriptor for given addresses when asked for a pool corresponding to a specific address */
 	MM_LightweightNonReentrantLock _expandLock; /**< Most of the common expand code is not multi-threaded safe (since it used in standard collectors on alloc path fail path which is single threaded)  */
+	double _lastObservedGcPercentage; /**< The most recently observed GC percentage (time gc is active / time gc is not active) */
+	U_64 _previouslyObservedPGCCount; /**< The count of observed PGC's since the last GMP cycle - Used to determine if heap resize occured after a system/global gc */
```

Comment:
if you change variable types to standard in correspondent `.cpp` file, so it make sense to change types in `.hpp` as well as a pair